apps/judicial-system/web/src/components/Tags/TagCaseState/TagCaseState.tsx

```
@@ -1,38 +1,35 @@
+  CaseListEntry,
   CaseState,
-  CaseType,
   Defendant,
   IndictmentDecision,
   User,
 } from '@island.is/judicial-system-web/src/graphql/schema'
 
+import { UserContext } from '../../UserProvider/UserProvider'
 import { strings } from './TagCaseState.strings'
 
+interface Tag {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Tag'. Consider to delete it or rename it.

'Tag' is defined here:


(lint/suspicious/noRedeclare)